catroid/src/main/java/org/catrobat/catroid/content/actions/PointToAction.java

```diff
@@ -27,23 +27,16 @@
 import org.catrobat.catroid.ProjectManager;
 import org.catrobat.catroid.content.Sprite;
 
-import java.util.ArrayList;
-
 public class PointToAction extends TemporalAction {
 
 	private Sprite sprite;
 	private Sprite pointedSprite;
 
 	@Override
 	protected void update(float percent) {
-		final ArrayList<Sprite> spriteList = (ArrayList<Sprite>) ProjectManager.getInstance().getSceneToPlay()
-				.getSpriteList();
-		if (!spriteList.contains(pointedSprite)) {
-			pointedSprite = null;
-		}
-
-		if (pointedSprite == null) {
-			pointedSprite = this.sprite;
+		if (pointedSprite == null
+				|| !ProjectManager.getInstance().getSceneToPlay().getSpriteList().contains(pointedSprite)) {
```

Comment:
IMO it's not the job of the PointToAction to check if the Scene contains the pointedSprite. However, I kept it.